### PR TITLE
Fixes banhammer grammar

### PR DESCRIPTION
**Pull request by Michael-Ashfield**
[Changelogs]: # Fixed grammar on the banhammer item when used on players heads.
Converts:
"[user] are stroking the head of [M] with a bangammer"
To:
"[user] is stroking the head of [M] with a banhammer"
Fixes and additional "Bangammer" to its correct spelling.

:cl: Mickyy5
spellcheck: Fixed grammar on the banhammer
/:cl:

[why]: # The banhammer item had incorrect grammar and spelling when being targeted on the head.

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");
+		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a banhammer</span>", "<span class='userdanger'>[user] is stroking the head with a banhammer</span>", "you hear a banhammer stroking a head");
```
Comment (SpaceManiac) [inline code/game/objects/items/weaponry.dm]: Use `[user] [user.p_are()] stroking`. It'll come out to "is" most of the time but remain "are" in the narrow cases where doing so is appropriate.

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (SpaceManiac) [reply code/game/objects/items/weaponry.dm]: Actually I'm not entirely sure about this, I'll take a deeper look at some point.

**Comment by optimumtact** [comment]
This is a meme relating to the english translation of the ss13 russian wiki page on lurkmore.

This is now the second pr fixing this so at this point I don't even care to keep it

**Comment by SpaceManiac** [comment]
If it's intended add a comment so people know that and move on

**Comment by MrDoomBringer** [comment]
Means that one of the crew members works for a criminal Syndicate and
must something to seize / kill someone / someone to protect and dump
from that Durkee on the shuttle. At English stations the word everyday,
for Russians for the mention of the word "traitor" is not in the chat
(IC) admins are stroking the head with a bangammer. So this word and
memetic.

**Comment by MrDoomBringer** [comment]
[mfw newfags dont bangammer](https://github.com/tgstation/tgstation/pull/30173)

**Comment by ninjanomnom** [comment]
Well I'm certainly fine doing that especially seeing as they haven't responded to anything in here. If someone wants to add a comment in another pr, go for it.

**Pull request closed without merging**